spec/models/dialog_field_drop_down_list_spec.rb

```diff
@@ -236,6 +236,35 @@
         end
       end
 
+      context "dialog field dropdown without options hash" do
+        before do
+          @df = FactoryGirl.create(:dialog_field_drop_down_list, :name => 'test drop down')
+        end
+
+        it "#multi_value?" do
+          expect(@df.multi_value?).to be_falsey
+
+          @df.force_multi_value = true
+          expect(@df.multi_value?).to be_truthy
+        end
+
+        it "#automate_key_name" do
+          expect(@df.automate_key_name).to eq("dialog_#{@df.name}")
+        end
+      end
+
+      context "dialog field dropdown with options hash" do
+        before do
+          @df = FactoryGirl.create(:dialog_field_drop_down_list,
+                                   :name    => 'test drop down',
+                                   :options => {:force_multi_value => true})
+        end
+
+        it "#multi_value?" do
+          expect(@df.multi_value?).to be_truthy
+        end
+      end
+
```

Comment:
I think you need a few more specs to cover the fact that you're checking 3 things in the actual `multi_value?` method. Maybe structured like this?:
```
describe "#multi_value?" do
  context "when force_multi_value is present" do
    context "when force_multi_value is null" do
      # it "returns false"
    end

    context "when force_multi_value is not null" do
      context "when force_multi_value is truthy" do
        # it "returns true"
      end

      context "when force_multi_value is falsy" do
        # it "returns false"
      end
    end
  end

  context "when force_multi_value is not present" do
    # it "returns false"
  end
end